webdriver/tests/bidi/browsing_context/capture_screenshot/capture_screenshot.py

```diff
@@ -35,3 +36,25 @@ async def test_capture(bidi_session, top_context, inline, compare_png_bidi):
 
     comparison = await compare_png_bidi(new_data, data)
     assert comparison.equal()
+
+@pytest.mark.asyncio
+async def test_capture_with_viewport(bidi_session, new_tab, inline, compare_png_bidi):
```

Comment:
Probably we should parameterize this test to have a smaller and a larger size for the viewport compared to the original values.

Reply:
Added